src/Tribe/Views/V2/Hooks.php

```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
 		$assets->disable_v1();
 	}
 
+	public function action_register_widget_styles() {
+		$style_dir = plugin_dir_url( TRIBE_EVENTS_FILE ) . 'src/resources/css/';
+
+		wp_register_style(
+			'tribe-events-widgets-v2-skeleton',
+			$style_dir . 'widgets-skeleton.css',
+			[ 'tribe-common-skeleton-style' ],
+		);
+
+		wp_register_style(
+			'tribe-events-widgets-v2-full',
+			$style_dir  . 'widgets-full.css',
```

Comment:
:no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: 849b337279e60425df30c92fe3c0032c</sub>